indico/modules/events/registration/controllers/management/__init__.py

```diff
@@ -52,3 +53,23 @@ def _process_args(self):
                              .options(defaultload(Registration.data)
                                       .joinedload('field_data'))
                              .one())
+
+
+class RHManageRegistrationFieldActionBase(RHManageRegFormBase):
```

Comment:
Do you propose a better name for this base RH?

Reply:
let's keep that name for now, if i come up with a nicer one during review i can always change it